src/OpenTelemetry/.publicApi/Experimental/PublicAPI.Unshipped.txt

```diff
@@ -12,16 +12,38 @@ OpenTelemetry.Metrics.AlwaysOnExemplarFilter.AlwaysOnExemplarFilter() -> void
 OpenTelemetry.Metrics.Exemplar
 OpenTelemetry.Metrics.Exemplar.DoubleValue.get -> double
 OpenTelemetry.Metrics.Exemplar.Exemplar() -> void
+OpenTelemetry.Metrics.Exemplar.FilteredTags.get -> OpenTelemetry.ReadOnlyFilteredTagCollection
+OpenTelemetry.Metrics.Exemplar.LongValue.get -> long
 OpenTelemetry.Metrics.Exemplar.SpanId.get -> System.Diagnostics.ActivitySpanId?
 OpenTelemetry.Metrics.Exemplar.Timestamp.get -> System.DateTimeOffset
 OpenTelemetry.Metrics.Exemplar.TraceId.get -> System.Diagnostics.ActivityTraceId?
 OpenTelemetry.Metrics.ExemplarFilter
 OpenTelemetry.Metrics.ExemplarFilter.ExemplarFilter() -> void
-OpenTelemetry.Metrics.MetricPoint.GetExemplars() -> OpenTelemetry.Metrics.Exemplar[]!
+OpenTelemetry.Metrics.ExemplarMeasurement<T>
```

Comment:
Could we mark this internal? Do we expect users to use `ExemplarMeasurement` in their code?

Reply:
Ahh just noticed that this used as part of the Exemplar reservoir contract.

Reply:
`ExemplarReservoir` is `internal` so we _could_ make `ExemplarMeasurement` `internal` BUT the spec says we need to expose `ExemplarReservoir` on the View API (follow-up PR) so it is likely going to end up public anyway 😄

Reply:
Agree. lets keep it public as the next set of PRs need it anyway